rules_typescript_2.patch

```diff
@@ -0,0 +1,70 @@
+diff internal/tsc_wrapped/tsconfig.ts internal/tsc_wrapped/tsconfig.ts
+index be964eb..edd6b30 100644
+--- internal/tsc_wrapped/tsconfig.ts
++++ internal/tsc_wrapped/tsconfig.ts
+@@ -201,6 +201,17 @@ export interface BazelOptions {
+    * future.
+    */
+   devmodeTargetOverride?: string;
++
++  /**
++   * Override for module format to use for devmode.
++   *
++   * This setting can be set in a user's tsconfig to override the default
++   * devmode module.
++   *
++   * EXPERIMENTAL: This setting is experimental and may be removed in the
++   * future.
++   */
++  devmodeModuleOverride?: string;
+ }
+ 
+ export interface ParsedTsConfig {
+@@ -285,6 +296,10 @@ export function parseTsconfig(
+           devmodeTargetOverride: isUndefined(existingBazelOpts.devmodeTargetOverride)
+             ? newBazelBazelOpts.devmodeTargetOverride
+             : existingBazelOpts.devmodeTargetOverride,
++
++          devmodeModuleOverride: isUndefined(existingBazelOpts.devmodeModuleOverride)
++            ? newBazelBazelOpts.devmodeModuleOverride
++            : existingBazelOpts.devmodeModuleOverride,
+         }
+       }
+ 
+@@ -359,6 +374,36 @@ export function parseTsconfig(
+   // the generated tsconfig indicates otherwise.
+   if (bazelOpts.googmodule) options.module = ts.ModuleKind.CommonJS;
+ 
++  // If not googmodule, override the devmode module if devmodeModuleOverride is set
++  else if (bazelOpts.es5Mode && bazelOpts.devmodeModuleOverride) {
++    switch (bazelOpts.devmodeModuleOverride.toLowerCase()) {
```

Comment:
hmm, isn't there a way to convert the option without us having to enumerate all of them?

Reply:
none that I know of. devmodeModuleTarget was enumerated as well